lib/identity_doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -18,24 +19,42 @@ def initialize(http_response, liveness_checking_enabled, config)
 
         def successful_result?
           transaction_status_passed? &&
+            true_id_product.present? &&
             product_status_passed? &&
             doc_auth_result_passed?
         end
 
         def error_messages
           return {} if successful_result?
 
-          ErrorGenerator.new(config).generate_trueid_errors(response_info, @liveness_checking_enabled)
+          if true_id_product.present?
+            ErrorGenerator.new(config).generate_trueid_errors(response_info, @liveness_checking_enabled)
+          else
+            { network: true } # return a generic technical difficulties error to user
+          end
         end
 
         def extra_attributes
-          attrs = response_info.merge(true_id_product[:AUTHENTICATION_RESULT])
-          attrs.reject do |k, _v|
-            PII_DETAILS.include? k
+          if true_id_product.present?
+            attrs = response_info.merge(true_id_product[:AUTHENTICATION_RESULT])
+            attrs.reject do |k, _v|
+              PII_DETAILS.include? k
+            end
+          else
+            response_status = {
+              LN_Status: parsed_response_body[:Status],
+              LN_Info: parsed_response_body.dig(:Information)
```

Comment:
how do you feel about making these `snake_case` keys to match what we do elsewhere? I know thjis gem already uses some `CamelCase` but this seems like it's mixing style? Also no need to abbreviate? `lexis_nexis_status`, etc?

Reply:
Yeah, sorry. UpperCamelCase is so much my default that I tried to do it correctly but ended up mixing the two. I'll fix it before I merge.